integration/loader/main.go

```diff
@@ -193,6 +192,8 @@ func main() {
 				lg.Stop()
 			}
 		}
+
+		cancel()
```

Comment:
why do we need to run this big loop in a goroutine?  can you simplify this to just run directly in the main function?

I think we can completely remove the context.  it doesn't provide any value since it never times out, and is only cancel here.

Reply:
If we remove the context we lose in-line logging and prometheus data. cancel() is called here to prevent a deadlock that existed previously.

Reply:
Simplified loop down to single case that will be used in CI, the subroutine is still required to allow for inline logging and pushing of data to local Prometheus